### ISSUE TITLE
[Request]: Add a more elaborated priority system

### ISSUE THREAD

**Issue by resucutie** [dgsasha/remembrance#44]
### Tell us the problem or your need

As of now, Reminders' current task priority system is based on Microsoft To-Do's priority system, where you add a star on important tasks. The issue is that this system is incompatible with how CalDAV's priority system works, more specifically the Nextcloud Tasks one, that is a number ranging from 0 to 9, where 0 is "no priority", 1-4 is "high priority", 5 "medium priority" and 6-9 "low priority". This can override existing priorities due to how Reminders interprets priorities, that is, high priority is "important", everything else is not. Here's an image showing that:
![image](https://github.com/remindersdevs/Reminders/assets/27961224/dcc2fa02-80c5-4421-9b2a-d910e6b0374a)

### Describe the solution you'd like

The way that it could be fixed would be to do what Nextcloud Tasks does, that is, a slider that goes from 0 to 9.
![image](https://github.com/remindersdevs/Reminders/assets/27961224/9781c860-52c0-4bcd-b488-ba229b5eee60)
For syncing with Microsoft To-Do, you could either simply mark everything from 1-4 as "important" and everything else as "not important". Maybe for To-Do tasks you could replace the slider with the current switcher for simplicity's sake.

As for how it should be shown in the lists UI, the star icon could become colored depending of the priority. Example: 0 wouldn't show the star, 1-4 would show it red, 5 yellow, and 6-9 blue.

### Describe alternatives you've considered

Another solution would be to do something similar to what Tasks.org does, that is, show options that the user could mark. This would be more user friendly as this explains the priority system with just a simple look  
![Screenshot_20230514_114644_Tasks](https://github.com/remindersdevs/Reminders/assets/27961224/fa09a549-e128-4812-9157-48b158fc1519)

**Comment by dgsasha** [comment]
>Another solution would be to do something similar to what Tasks.org does, that is, show options that the user could mark. This would be more user friendly as this explains the priority system with just a simple look

I like this.

**Comment by resucutie** [comment]
yeah i also agree. seems a better way to handle that, although kinda limiting since you cannot put values from 0 to 9. maybe adding a text input after the selectors?
